public/angular/js/services/map.js

```diff
@@ -42,13 +42,17 @@ angular.module('Aggie')
       google.maps.event.addListener(m, 'click', function() { self.show_info_window(this); });
     });
 
-    if (self.markers.length == 0)
+    if (self.markers.length == 0) {
       self.map.setCenter(new google.maps.LatLng(0, 0));
-
-    // else use bounds determined above
-    else
+    } else if (self.markers.length === 1) {
```

Comment:
That's a much better idea :)  Okay, doing fitBounds, adjusting if zoom is too high. There was some weirdness with fitBounds (it is an async fn), but found fix on stackoverflow, seems to work great.